frontend/src/component/Account/PersonalData.jsx

```diff
@@ -63,25 +64,31 @@ export class PersonalData extends React.Component {
 
         <form className='personaldata__form'>
           <div className='d-flex align-items-center flex-wrap mb-4'>
-            <input
-              className='personaldata__form__txtinput primaryColorBorderLighten form-control'
-              type='text'
-              data-cy='personaldata__form__txtinput__fullname'
-              placeholder={props.t('New full name')}
-              value={state.newPublicName}
-              onChange={this.handleChangePublicName}
-            />
+            <label>
+              {props.t('New full name')}
```

Comment:
```suggestion
              {props.t('New full name:')&nbsp;
```
you will need to update the translation strings too